packages/cli/src/utils/string.js

```diff
@@ -0,0 +1,13 @@
+function splitFileFromPath(filePath, extension = 'js') {
+  const destParts = filePath.split('/');
```

Comment:
🚨 I'd use `path.separator` here, for cross-platform compatibility. Same below.

Reply:
Good call!